docs/timezone.md

```diff
@@ -146,10 +149,10 @@ When subclassing `Temporal.TimeZone`, this property doesn't need to be overridde
 
 **Returns:** The UTC offset at the given time, in nanoseconds.
 
-Since the UTC offset can change throughout the year in time zones that employ DST, this method queries the UTC offset at a particular time.
+Since the UTC offset can change throughout the year in time zones that employ DST as well as because of special political decisions, this method queries the UTC offset at a particular time.
 
-Note that only `Temporal.TimeZone` objects constructed from an IANA time zone name may have DST transitions; those constructed from a UTC offset do not.
-If `timeZone` is a UTC offset time zone, the return value of this method is always the same regardless of `instant`.
+Note that only `Temporal.TimeZone` objects constructed from an IANA time zone name may have UTC offset transitions; those constructed from a UTC offset are fixed.
```

Comment:
I agree with your changes. While reading the original text, I think the original text could be clearer. So on top of your changes, would it be worth clarifying the text further? Maybe something like this to replace this and the sentence below it?

```suggestion
Note that `Temporal.TimeZone` objects constructed from an IANA time zone name may change offsets depending on the instant.
However, other time zones (some IANA time zones like `Etc\GMT+5` and all time zones constructed from a numeric UTC offsets) have fixed offsets that never change, regardless of the instant.
```